helm/cas-airflow-dag-trigger/values.yaml

```diff
@@ -0,0 +1,12 @@
+# Values
+enable: true
```

Comment:
is this needed?

Reply:
not really, but it's nice to have a quick off switch as a dev? I'd be in favour of keeping it

Reply:
You can have conditional dependencies in helm charts: https://helm.sh/docs/chart_best_practices/dependencies/#conditions-and-tags, so I don't think the flag is needed in the chart itself